src/theme/css-themes/css-variables.ts

```diff
@@ -100,12 +100,83 @@ const BOOSTED5_VARIABLE = `
 [data-bs-theme] .ods-charts-context a, .ods-charts-context[data-bs-theme] a {
     color: var(--bs-link-color);
 }
+    
+.ods-charts-context {
+  --bs-body-font-weight: 400;
+}
+`;
+
+/**
+ * Added for None theme
+ */
+const NONE_THEME_SPECIFIC_VARIABLES = `
+.ods-charts-context {
+  --bs-font-sans-serif:  "Helvetica Neue", Helvetica, "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+`;
+
+/**
+ * Added for Boosted 4 theme
+ */
+const BOOSTED4_THEME_SPECIFIC_VARIABLES = `
+.ods-charts-context {
+  --bs-font-sans-serif:  var(--font-family-sans-serif, "Helvetica Neue");
+}
+
+.ods-charts-context .popover {
+  color: var(--bs-body-color);
+  background-color: var(--bs-body-bg);
+  border: 1px solid var(--bs-border-color-subtle);
```

Comment:
In Boosted 5, it's 2px border:
```suggestion
  border: 2px solid var(--bs-border-color-subtle);
```